build.yaml

```diff
@@ -1242,7 +1236,6 @@ steps:
      export HAIL_DOCTEST_DATA_DIR=./data
      export HAIL_TEST_BUCKET=hail-test-dmk9z
      export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
-     export PYSPARK_SUBMIT_ARGS="--driver-memory 6g pyspark-shell"
```

Comment:
I'm assuming it's a mistake that only `test_hail_python_local_backend_2` still has HAIL_TEST_BUCKET and GOOGLE_APPLICATION_CREDENTIALS?